_pages/sponsors.md

```diff
@@ -7,13 +7,17 @@ permalink: /sponsors/
 
 <div class="pretty-links">
 
-We're currently looking for sponsorship! More information to be posted shortly. If you're interested, email us at starconuw@googlegroups.com
+We're currently looking for sponsors!
 
-<!--
+## Why You Should Sponsor Us
 
-TODO: once Arshia writes her email we can put a bunch of that stuff here
+We decided to organize StarCon because we want to invite people excited about the work they do to share that excitement with others--because geniune enthusiasm is inspiring! We want to organize an event in which anyone interested about technology, regardless of their experience level, would feel motivated to pursue that interest. We want to keep this event free, encourage students to speak and attend, provide an accessible venue, make sure our attendees are well fed, and give them some swag to remember us by!
 
--->
+If you're interested in sponsoring us, or have any questions about sponsorship, email us!
+
+## Contact
+
+Arshia Mufti: [amufti16@gmail.com](mailto:amufti16@gmail.com)
```

Comment:
Add that you're director of sponsorship maybe? Idk if the title would look more legit than just a name